src/ensembl/src/content/app/entity-viewer/gene-view/GeneView.tsx

```diff
@@ -314,7 +315,7 @@ const GeneViewWithData = (props: GeneViewWithDataProps) => {
         <div className={styles.tabWrapper}>
           <GeneViewTabs isFilterOpen={isFilterOpen} />
         </div>
-        {isFilterOpen && (
+        {props.gene.transcripts.length > 5 && isFilterOpen && (
```

Comment:
`props.gene.transcripts.length > 5` looks very suspicious here. If the redux state, for whatever reason, thinks that the filter box should be open, then it should be open. `props.gene.transcripts.length` should only dictate whether we see the button that can expand the filter box

Reply:
With the old condition which was only checking for `isFilterOpen` this has now caused a bug to appear. since the `isFilterOpen` is now set in redux and if it is set to true (from another species/gene), it will show the panel even if we are not showing the filter link. Probably both should be wrap in the same condition (the link and the panel).

Reply:
> isFilterOpen is now set in redux and if it is set to true (from another species/gene), it will show the panel even if we are not showing the filter link

Umm, this sounds like a flaw in the code. Needs some thought.